source/particle/property/cpo_elastic_tensor.cc

```diff
@@ -101,6 +101,10 @@ namespace aspect
               {
                 stiffness_matrix = &stiffness_matrix_enstatite;
               }
+            else if (cpo_particle_property.get_deformation_type(cpo_data_position,data,mineral_i) == DeformationType::clinopyroxene)
+              {
+                // skip, CPX stiffness_matrix not developed yet
```

Comment:
Should we then crash if we ever get into this if-branch? Otherwise we return wrong values from this function. You could do:
```suggestion
                AssertThrow(false, ExcMessage("This functionality is not implemented yet.");
```

Reply:
This is to prevent the next else condition that lead to crashing when having deformationtype of clinopyroxene:
`else
              {
                AssertThrow(false, ExcMessage("Stiffness matrix not implemented for deformation type "
                                              + std::to_string(static_cast<unsigned int>(cpo_particle_property.get_deformation_type(cpo_data_position,data,mineral_i)))));
              }`